nrich-spring-util/build.gradle

```diff
@@ -1,10 +1,10 @@
 dependencies {
   annotationProcessor "org.projectlombok:lombok"
-
```

Comment:
Not sure about this removal. It seems better with a new line to me.

Reply:
I was just syncing it with the rest of the project, since this is the only place that has them separated.

Reply:
If this approach is currently one we use across the project then we can keep it here. 
Maybe we can change this in a separate issue later if we agree that it's better with a new line.

Related to the other part, I don't think that we should group them just because of that.

Reply:
We can have a separate issue if we agree to change, this was out of sync with rest of the files thats why it is changed here.